### ISSUE TITLE
Function values do not work with arrays

### ISSUE THREAD

**Issue by lttb** [cssinjs/jss#497]
[Online example](https://www.webpackbin.com/bins/-KkQSwYoUGrwiKYSP_wY)

it does not work:
```jsx
const styles = {
  title: {
    fontSize: 20,
    border: ({ border }) => border && [2, 'solid', 'red']
  }
}
```

but this works:

```jsx
const styles = {
  title: {
    fontSize: 20,
    border: ({ border }) => border && ['2px', 'solid', 'red'].join(' ')
  }
}
```

**Comment by kof** [comment]
So the dilemma here is which plugins should react on onChangeValue, more of them will increase DX but lower performance. The problem with perf here is that this hook is called when function values change, which might be used in a high performance scenario like an animation. Any overhead added there would mean overhead to every animation frame.

**Comment by kof** [comment]
fixed in 7.1.3

**Comment by kof** [comment]
wrong issue

**Comment by kof** [comment]
released in v9.4.0
